tidb-cloud/data-service-get-started.md

```diff
@@ -131,9 +133,31 @@ You can customize SQL statements for the endpoint in the SQL editor, which is th
 
     If you have inserted parameters in the SQL statements, make sure that you have set test values or default values for the parameters in the **Params** tab on the right pane. Otherwise, an error is returned.
 
-    To run a SQL statement, select the line of the SQL with your cursor and click **Run** > **Run at cursor**.
+    <SimpleTab>
+    <div label="macOS">
+
```

Comment:
Tip for reviewers:

Line 136~L160 are the same with that in  https://github.com/pingcap/docs/blob/release-7.5/tidb-cloud/data-service-manage-endpoint.md?plain=1#L158-L182